docs/index.md

```diff
@@ -0,0 +1,47 @@
+# Song Developer Documentation
+
+---
+
+**Navigation**
+- [Operation](./operation/operation.md)
+- [Contribution](./contribution/contribution.md) 
+
+---
+
+# Background
+
+Song is a metadata validation and tracking tool designed to streamline the management of genomics data across multiple cloud storage systems. With Song, users can create high-quality and reliable metadata repositories with minimal human intervention. As a metadata management system, Song does not handle file transfer and object storage. Song interacts with a required companion application, <a href="https://github.com/overture-stack/score" target="_blank" rel="noopener noreferrer">Score</a>, which manages file transfers and object storage.
+
+## Data Submission
+
+**Analysis Files:** An analysis is a description of a set of one or more files plus the metadata describing that collection of files.
+
+**Metadata Validation:** Analyses get validated against the administrator's Dynamic Schema. That defines the vocabulary and structure of the analysis document. 
```

Comment:
```suggestion
**Metadata Validation:** Analyses get validated against a metadata schema that defines the vocabulary and structure of the analysis document. Song allows administrators to define custom schemas that describe the Analyses they intend to manage.
```

Not sure on the wording of my second suggested sentence... what i'm trying to do is indicate that you can define multiple schemas, one for each analysis type that you will be managing. The phrasing "the administrator's Dynamic Schema" seems to imply that the administrator only gets one custom schema

Reply:
Your wording here is better.